gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -1398,7 +1398,8 @@ private static void checkContentType(BlobInfo blobInfo) throws IllegalArgumentEx
   byte[] readAllBytes(BlobId blob, BlobSourceOption... options);
 
   /**
-   * Send a batch request.
+   * Send a batch request. If the {@code batchRequest} parameter contains more than 100 deletes the
```

Comment:
Yes, please lets take it out.

I think that in general we should not commit in our API to the number of RPCs that we are actually going to perform per API call and in fact it may even changed per platform. (e.g. to support AE URL request [limits] (https://cloud.google.com/appengine/docs/java/urlfetch/#Java_Quotas_and_limits)).

I think one of the goals of our API it to perform a user request in an optimal way and applying
this chunking is doing exactly that.